### PR TITLE
Docs: Update README.md for FormToggle component 

### PR DESCRIPTION
**Pull request by imangm**
State variable and function name were invalid.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: packages/components/src/form-toggle/README.md [hunk 1]
```diff
@@ -61,8 +61,8 @@ const MyFormToggle = () => {
 	const [ isChecked, setChecked ] = useState( true );
 
 	<FormToggle
-		checked={ checked }
-		onChange={ () => setChecked( ( state ) => ! state ) }
+		checked={ isChecked }
+		onChange={ () => setIsChecked( ( state ) => ! state ) }
```
Comment (Mamaduka) [inline packages/components/src/form-toggle/README.md]: Thanks for contributing, @imangm. The `setChecked` is the correct function name. The only variable name was incorrect.

Comment (imangm) [reply packages/components/src/form-toggle/README.md]: > 

Thank you! I forgot to update the setter function name in the state. Isn't it better that we change the function name both in the state declaration and in onChange method to setIsChecked?

Comment (Mamaduka) [reply packages/components/src/form-toggle/README.md]: 100%. It will match the general `useState` naming convention :)

Comment (gziolo) [inline packages/components/src/form-toggle/README.md]:
```suggestion
		onChange={ () => setChecked( ( state ) => ! state ) }
```

**Review by gziolo — APPROVED** [review]
Thank you 👍

**Pull request merged**